apps/judicial-system/web/src/utils/hooks/useSections/index.ts

```diff
@@ -899,8 +900,9 @@ const useSections = (
     return {
       name: formatMessage(sections.indictmentsCourtSection.title),
       isActive:
-        (isDistrictCourtUser(user) || isDefenceUser(user)) &&
-        !isCompletedCase(state),
+        (isProsecutionUser(user) && state === CaseState.RECEIVED) ||
```

Comment:
Consider renaiming this method to something other than `getIndictmentsCourtSections` as it is also used for prosecutors. Or is this a mistake?

Also consider checking that state === CaseState.MAIN_HEARING as that state can also occur during the court proceedings.

Reply:
It's not really used by prosecutors, per se. It's the court section, i.e. "Héraðsdómur RVG" f.x. Everyone sees this section but in this case we are deciding whether or not it's active or not.